web/src/main/java/com/google/sps/servlets/SaveStockServlet.java

```diff
@@ -47,6 +49,41 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     long timestamp = System.currentTimeMillis();
 
+    Date date = Calendar.getInstance().getTime();
+    String gooddate = date.toString();
+    String[] date_ok = gooddate.split(" ");
```

Comment:
use camel case dateOk (also this should be more descriptive than "date ok". If the other people on the team can't tell me what this variable is suppose to mean based on the name then it isn't descriptive enough. Same with goodDate above.

Reply:
right right